lib/pbench/agent/base.py

```diff
@@ -30,20 +28,19 @@ def __init__(self, context):
         # the pbench temporary directory is always relative to the $pbench_run
         # directory
         self.pbench_tmp = self.pbench_run / "tmp"
-        try:
-            fs.safe_mkdir(self.pbench_tmp)
-        except Exception:
-            click.secho(f"[ERROR] unable to create TMP dir, {self.pbench_tmp}")
-            sys.exit(1)
+        if not self.pbench_tmp.exists():
+            try:
+                os.makedirs(self.pbench_tmp)
+            except OSError:
+                click.secho(f"[ERROR] unable to create TMP dir, {self.pbench_tmp}")
+                sys.exit(1)
 
         # log file - N.B. not a directory
-        self.pbench_log = self.get_path(os.environ.get("pbench_log", None))
+        self.pbench_log = self.config.pbench_log
         if self.pbench_log is None:
             self.pbench_log = self.config.pbench_log
 
-        self.pbench_install_dir = self.get_path(
-            os.environ.get("pbench_install_dir", None)
-        )
+        self.pbench_install_dir = self.config.pbench_install_dir
```

Comment:
Same thing here: I set it to something and if it is None, I set it to the same something?

Reply:
fixed